<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>500</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QRadioButton" name="btnAnnuity">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Аннуитетный</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnDiff">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>130</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Дифференцированный</string>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>200</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="creditAmountTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Сумма кредита</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Срок</string>
   </property>
  </widget>
  <widget class="QLabel" name="percentageTitle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Процентная годовая ставка</string>
   </property>
  </widget>
  <widget class="QLabel" name="paymentTypeLabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>20</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Тип выплат:</string>
   </property>
  </widget>
  <widget class="QLabel" name="overallPaymentTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Общая выплата:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overallPay">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>310</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="monthlyPay">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>370</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="monthlyPaymentTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Ежемесячный платеж</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="overpayment">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>430</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="overpaymentTitle">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>420</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Переплата</string>
   </property>
  </widget>
  <widget class="QComboBox" name="timeCombobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1 месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 месяца</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6 месяцев</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1 год</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4 года</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15 лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20 лет</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="percentageCombobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>25.0</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="creditAmountLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QSlider" name="creditAmountSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>261</width>
     <height>25</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>24</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="rubleLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>50</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>₽</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
